tailcfg/tailcfg.go

```diff
@@ -345,15 +345,17 @@ func (id GroupID) String() string      { return fmt.Sprintf("groupid:%x", int64(
 func (id RoleID) String() string       { return fmt.Sprintf("roleid:%x", int64(id)) }
 func (id CapabilityID) String() string { return fmt.Sprintf("capid:%x", int64(id)) }
 
+// Equal returns whether n and n2 serialize into an equivalent byte array.
```

Comment:
Go styles for bool funcs is to say "reports whether",  not "returns whether". (Argument from Rob Pike originally was that: "it always returns")

Also, no need to mention implementation details in the doc. The point is just that they're equal, not how we check that.

Reply:
See what I did in control/controlclient/controlclient_test.go and control/controlclient/auto.go for Status.Equal.